fritzconnection/cli/fritzwol.py

```diff
@@ -0,0 +1,73 @@
+"""
+fritzwol.py
+
+Module to wake up a single host via the Fritzbox built-in mechanism.
+This can be helpful if the host to be woken up is in a different
+broadcast domain/ subnet than the client which tries to wake up.
+CLI interface.
+
+This module is part of the FritzConnection package.
+https://github.com/kbr/fritzconnection
+License: MIT (https://opensource.org/licenses/MIT)
+Author: Maik Töpfer
+"""
+
+from fritzconnection.core.exceptions import FritzConnectionException
+from fritzconnection.core.fritzconnection import FritzConnection
+from ..lib.fritzhosts import FritzHosts
+from . utils import get_cli_arguments, get_instance, print_header
+
+
+class WakeOnLan:
+    def __init__(self, fc, fh):
+        self._fc = fc
+        self._fh = fh
+
+    def wakeup(self, host_name):
+        print(f"Waking up host '{host_name}'...")
```

Comment:
Please don't use `print()`in code not supposed to interact with the user. `main()`is the function where print should get used.